sdk/keyvault/azure-security-keyvault-jca/pom.xml

```diff
@@ -0,0 +1,232 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <artifactId>azure-client-sdk-parent</artifactId>
+        <groupId>com.azure</groupId>
+        <version>1.7.0</version> <!-- {x-version-update;com.azure:azure-client-sdk-parent;current} -->
+        <relativePath>../../parents/azure-client-sdk-parent</relativePath>
+    </parent>
+
+    <groupId>com.azure</groupId>
+    <artifactId>azure-security-keyvault-jca</artifactId>
+    <version>1.0.0-beta.1</version> <!-- {x-version-update;com.azure:azure-security-keyvault-jca;current} -->
+    <name>JCA Provider for Azure Key Vault</name>
+    <description>The Java Crypto Architecture (JCA) Provider for Azure KeyVault</description>
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <version>3.8.1</version> <!-- {x-version-update;org.apache.maven.plugins:maven-compiler-plugin;external_dependency} -->
+                <configuration>
+                    <source>1.8</source>
+                    <target>1.8</target>
```

Comment:
Because we want this library to work on Java 8.

Reply:
okay, 
but we don't lock the target version here in our client sdk parent pom.
This can be cleaned up using profiles when CI is added for this module.

Reply:
[Yes we do](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/parents/azure-client-sdk-parent/pom.xml#L950)